x-pack/plugins/security_solution/public/common/translations.ts

```diff
@@ -19,6 +19,11 @@ export const BETA = i18n.translate('xpack.securitySolution.pages.common.beta', {
   defaultMessage: 'Beta',
 });
 
+export const BETA_TOOLTIP = i18n.translate('xpack.securitySolution.pages.common.beta.tooltip', {
+  defaultMessage:
+    'This functionality is in beta and may be changed or removed completely in a future release. Elastic will work to fix any issues, but features in beta are not subject to the support SLA of official GA features.',
```

Comment:
@caitlinbetz @dasansol92 @paul-tavares This needs an update.

Reply:
Updated it based on https://www.elastic.co/guide/en/kibana/8.12/infrastructure.html#infrastructure